osgar/tools/log2map.py

```diff
@@ -20,16 +20,28 @@ def main():
     parser.add_argument('logfile', help='recorded log file')
     parser.add_argument('--out', help='map output file')
     parser.add_argument('--pose2d', help='stream ID for pose2d messages', default='app.pose2d')
+    parser.add_argument('--camera', help='stream ID with image data')
     args = parser.parse_args()
 
     out_filename = args.out
     if out_filename is None:
-        out_filename = os.path.join(os.path.dirname(args.logfile), 'world.jpg')
+        out_filename = os.path.join(os.path.dirname(args.logfile),
+                                    os.path.basename(args.logfile).split('.')[0] + '.jpg')
```

Comment:
So maybe `os.path.splitext(args.logfile)[0] + '.jpg'` is what we want here?

Reply:
cool, I did not know this function -> Yes :)